crates/merod/src/cli/run.rs

```diff
@@ -12,7 +12,10 @@ use crate::cli::RootArgs;
 
 /// Run a node
 #[derive(Debug, Parser)]
-pub struct RunCommand;
+pub struct RunCommand {
+    #[arg(long)]
+    pub auth: bool,
```

Comment:
```suggestion
    #[arg(long, default)]
    pub auth: bool,
```

so it's not a required flag